### ISSUE TITLE
[BUG] It's seem not support laravel v.10

### ISSUE THREAD

**Issue by eric-young-foo** [mateusjunges/laravel-kafka#209]
**Describe the bug**
It's seem not support laravel v.10

**To Reproduce**
1. INSTALL LARAVELV10
2. SAIL UP ( SUPPORT RDKAFKA )
3. COMPOSER INSTALL

**Expected behavior**
SUCCESS INSTALL LIBRARY

**Screenshots**
<img width="2125" alt="image" src="https://github.com/mateusjunges/laravel-kafka/assets/137266087/f4c8ebf2-a1f1-4cda-8f53-d30d3ce61426">

**Desktop (please complete the following information):**
 - OS: Mac Studio
 - Browser: not use
 - Version: latest

**Smartphone (please complete the following information):**
 - Device: not use
 - OS: not use
 - Browser not use
 - Version not use

**Additional context**

I meet problem
```
root@d34309430339:/var/www/html# composer require mateusjunges/laravel-kafka
./composer.json has been updated
Running composer update mateusjunges/laravel-kafka
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - mateusjunges/laravel-kafka[v1.11.0, ..., v1.13.2] require flix-tech/avro-serde-php ^1.7 -> satisfiable by flix-tech/avro-serde-php[1.7.0, 1.7.1, 1.7.2].
    - mateusjunges/laravel-kafka[v1.0.0, ..., v1.5.3] require ext-rdkafka ^5.0|^4.0 -> it has the wrong version installed (6.0.3).
    - mateusjunges/laravel-kafka[v1.4.6, ..., v1.10.2] require monolog/monolog ^2.3 -> found monolog/monolog[2.3.0, ..., 2.9.1] but the package is fixed to 3.4.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - flix-tech/avro-serde-php[1.7.0, ..., 1.7.2] require guzzlehttp/promises ^1.4.0 -> found guzzlehttp/promises[1.4.0, ..., 1.5.3] but the package is fixed to 2.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires mateusjunges/laravel-kafka * -> satisfiable by mateusjunges/laravel-kafka[v1.0.0, ..., v1.13.2].
```

**Comment by sqrooted** [comment]
Workaround until package dependencies update - with Laravel 10, try:

- Make sure supported version of rdkafka is installed (newest is 6.0)

```bash
# Instead of...
$ pecl install rdkafka

# ...do this
$ pecl install rdkafka-5.0.2
```

- Force re-install of `monolog/monolog` (dependency constraints in this package and laravel-framework conflict if laravel is installed first and monolog is composer.lock-d, but they are not mutually exclusive)

**Comment by DonovanChan** [comment]
@sqrooted were you actually able to install this with Laravel 10 that way? Laravel 10 has explicitly required monolog/monolog:^3.0 since it was [released](https://github.com/laravel/framework/blob/v10.0.0/composer.json). This is what I'm seeing:

```
$ rm composer.lock
$ composer require monolog/monolog:"^2.0"
Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update monolog/monolog
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/framework[v10.14.0, ..., v10.23.1] require monolog/monolog ^3.0 -> found monolog/monolog[3.0.0, ..., 3.4.0] but it conflicts with your root composer.json require (^2.0).
    - Root composer.json requires laravel/framework ^10.14 -> satisfiable by laravel/framework[v10.14.0, ..., v10.23.1].
```

**Comment by atlasag** [comment]
Hi Mateus, 
Looks like flix-tech/avro-serge-php ignoring your issue. ((( 
What about to switch dependency to your fork - mateusjunges/avro-serde-php  ?

**Comment by mateusjunges** [comment]
Ow I already did it when working on a feature for v2, but forgot to update for v1.13 and fix this. Working on it now

**Comment by mateusjunges** [comment]
@atlasag I just published `v1.13.8` which should contain the fix for this. Feel free to open another issue if the issue persists.
